nbgrader/docs/source/configuration/jupyterhub/nbgrader_config.py

```diff
@@ -36,11 +30,11 @@
 # the formgrader, then you can ignore this option.
 c.HubAuth.notebook_url_prefix = "path/to/class_files"
 
```

Comment:
In practice I needed to add the following option:

`c.HubAuth.grader_group = 'course1'`

I would add this and emphasize it has to match one of the groups created by JupyterHub